api/src/main/java/org/ehcache/expiry/Expirations.java

```diff
@@ -57,11 +57,24 @@
     return new TimeToIdleExpiry(timeToIdle);
   }
 
+  /**
+   * Builder allowing to specify TTL for creation, access and update. Unspecified values will be set to {@code Duration.INFINITE}
+   * meaning not expiration.
+   *
+   * @return an {@link Expiry} builder
+   */
+  public static <K, V> ExpiryBuilder<K, V> builder() {
```

Comment:
I am not a 100% convinced about having this builder made public. I understand it cleans up a lot of code in SPI tests, so that's a good thing.
But given the defaults to `Duration.INFINITE` I can't say I want it suddenly widely used.

Also if we wanted it public, it should live in a `.builder.` package.